3-platform/warehouse/spec/src/main/java/com/_6core/platform/warehouse/rest/v1/dto/product/RequestProductDescription.java

```diff
@@ -0,0 +1,4 @@
+package com._6core.platform.warehouse.rest.v1.dto.product;
+
+public record RequestProductDescription(Long productId, String description) {
```

Comment:
Remember about consistency in naming the same things -> https://github.com/sokimaaa/6Core/pull/34/files#diff-68e3389c734e0d8d9291a7033c39dd1517764fe154f3acce8c0facacb98a270f

Kateryna suggests placing in the end `Request` or `Response`. 
When I said that you need to review your teammate, The main goal was to make you see how your teammates doing the same things. And you staying consistent. 
```suggestion
public record ProductDescriptionRequest(Long productId, String description) {
```